app/web/features/dashboard/MyEvents.test.tsx

```diff
@@ -92,8 +92,8 @@ describe("My events", () => {
       expect(screen.getAllByRole("link")).toHaveLength(3);
       expect(listMyEventsMock).toHaveBeenCalledTimes(2);
       expect(listMyEventsMock.mock.calls).toEqual([
-        [{ pageSize: 3 }],
-        [{ pageToken: "2", pageSize: 3 }],
+        [{ pageSize: expect.any(Number) }],
+        [{ pageToken: "2", pageSize: expect.any(Number) }],
```

Comment:
```suggestion
        [{ pageSize: 2 }],
        [{ pageToken: "2", pageSize: 2 }],
```
I think it would be good to keep the exact `pageSize` assertion as before rather than say "any number". Otherwise if we change this in the future, we wouldn't notice something has changed and whether that change is intentional

Reply:
Sounds good! I'll make a variable in the test file, something like `cardPerRow` and use it here, so the number 2 has a meaning beyond being a hardcoded page size. Thanks!